js/src/views/Signer/containers/RequestsPage/requestsPage.js

```diff
@@ -114,7 +114,7 @@ class RequestsPage extends Component {
         title={
           <FormattedMessage
             id='signer.requestsPage.pendingTitle'
-            defaultMessage='Pending Requests'
+            defaultMessage='Pending Transactions'
```

Comment:
They are requests as in "requests for something to be signed". A transaction isn't valid until it's signed. But this interface is also used for more general "sign this random bit of data" as well. I agree that the terminology is a little muddy and clarification would lead to a better UX, but we also have to be careful to remain accurate.